odc/geo/crs.py

```diff
@@ -26,32 +29,54 @@
 
 SomeCRS = Union[str, int, "CRS", _CRS, Dict[str, Any]]
 MaybeCRS = Union[SomeCRS, Unset, None]
+EPSG_UNSET = 0
```

Comment:
To be safe, I am thinking `-1` may be a good choice.
```suggestion
EPSG_UNSET = -1
```